utils/check_repo.py

```diff
@@ -260,6 +260,7 @@ def get_model_modules():
         "modeling_tf_transfo_xl_utilities",
         "modeling_tf_vision_encoder_decoder",
         "modeling_vision_encoder_decoder",
+        "modeling_tf_swinv2",
```

Comment:
This shouldn't be added in case there's no TF implementation.

Reply:
After removing `test_modeling_tf_swinv2.py`, `make fixup` throws error. To solve this I just added it here, didn't found any other solution. Can you guide for same?

Reply:
You can run make style, make quality, make fix-copies separately.